src/MM2Helper.cpp

```diff
@@ -191,6 +191,9 @@ void MM2Helper::PreInit(const MM2Settings& settings, std::string* preset)
     }
     MapOpts.min_join_flank_ratio = 0.5;
 
+    //Allow for optional minimap2 'cs' tag
+    if (settings.OutCS) MapOpts.flag |= MM_F_OUT_CS;
```

Comment:
Setting the flag itself makes literally zero difference, as no part of the downstream code will react on it. `pbmm2` is more than just a mm2 wrapper.

Reply:
I'm sorry, I don't know where to change the mm2 flags if not here. If not here, where should I set the flag?

Reply:
You have to create the `cs` tag yourself, my first guess where to put it, is here: https://github.com/PacificBiosciences/pbmm2/blob/develop/src/MM2Helper.cpp#L673